pandas/io/json/_normalize.py

```diff
@@ -283,6 +371,14 @@ def _pull_records(js: Dict[str, Any], spec: Union[List, str]) -> List:
     else:
         raise NotImplementedError
 
+    if all(
```

Comment:
Good afternoon. Why do we need a `for` here? It seems to me that this option is cleaner: `all((record_prefix, meta, meta_prefix, record_prefix, max_level))`

Reply:
That is much simpler than my version, will change it to that at next opportunity

Reply:
after testing `any` is better here, more specifically 
```
not any((record_prefix, meta, meta_prefix, record_prefix, max_level))
```
as shown by
```
In [1]: t = (None, None)                             

In [2]: all(t)                                       
Out[2]: False

In [3]: t = (None, True)                             

In [4]: all(t)                                       
Out[4]: False

In [5]: t = (None, None)                             

In [6]: any(t)                                       
Out[6]: False

In [7]: t = (None, True)                             

In [8]: any(t)                                       
Out[8]: True
```